src/components/summerytable/summarytable-section.component.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from "react";
+import { useSelector } from "react-redux";
+import { ApplicationStateModel } from "../../state/models/application-state.model";
+import {
+  ScheduleLogicContext,
+  useScheduleState,
+} from "../schedule-page/table/schedule/use-schedule-state";
+import { SummaryTableRow } from "./summarytable-row.component";
+import { DataRow } from "../../logic/schedule-logic/data-row";
+
+export interface NameTableCellOptions {
+  dataRow: DataRow[];
+}
+
+export function SummaryTableSection({ dataRow }: NameTableCellOptions): JSX.Element {
+  const scheduleModel = useSelector((state: ApplicationStateModel) => state.scheduleData.present);
+
+  const { scheduleLogic, scheduleLocalState, setNewSchedule } = useScheduleState(scheduleModel);
+
+  useEffect(() => {
+    setNewSchedule(scheduleModel);
+  }, [scheduleModel, setNewSchedule]);
+
+  return (
+    <React.Fragment>
+      <table className="table" id="summaryTable">
+        <tbody>
+          <ScheduleLogicContext.Provider value={scheduleLogic}>
+            {dataRow.map((cellData) => {
+              return <SummaryTableRow uuid={scheduleLocalState.uuid} dataRow={cellData} />;
```

Comment:
Brakuje klucza dla elementów listy. 
Z tym jest taki problem, że w momencie kiedy w tym miejscu pojawią się nowe dane react nie będzie wiedział czy potrzeba renderować tą listę od nowa czy nie. 
https://reactjs.org/docs/lists-and-keys.html

Reply:
Można dodać coś w stylu `key={`${scheduleLocalState.uuid}_${cellData.rowKey}`}`